server/routes/index.js

```diff
@@ -281,74 +140,49 @@ router.get(
         subject: req.params.subject,
         defaultConfig,
       })
-      return res.send(
-        graphPage(
-          req.params.subject,
-          req.params.study,
-          req.user,
-          req.session.display_name,
-          req.session.icon,
-          req.session.mail,
-          req.session.toc,
-          dashboardState,
-          defaultConfig,
-          req.session.celery_tasks,
-          req.session.role
-        )
-      )
+      return res.status(200).json({
+        status: 200,
```

Comment:
```suggestion
```

Reply:
[b6abbbc](https://github.com/AMP-SCZ/dpdash/pull/319/commits/b6abbbc24c748cc38c461a04aec527463a8cb186)

Reply:
this comment still stands, the `status` field should be removed.

Reply:
[c247246](https://github.com/AMP-SCZ/dpdash/pull/319/commits/c2472461f08e7bd81bf1909aa22afd4c81658089)
Thanks for this